README.md

```diff
@@ -0,0 +1,31 @@
+
+One thing that is not clear is how many species of *Stereum* there are, how many are described, and how many have yet to be described. By amassing all the ITS sequence data available and creating a program to help me make sense of the data, I can estimate how many species there are and hopefully use metadata to pinpoint where on the globe they can be found, and if I am lucky, additional ecological data such as tree host species. 
+
+##### Materials and Methods
+*Data*
+* This project will use data acquired from NCBI GenBank in addition to 44 sequences that are included in my manuscript (https://www.biorxiv.org/content/10.1101/2020.10.16.342840v2) that have confirmed identities, to serve as references. 
+* My dataset as of **March 17, 2021** consists of a preliminary collection, to initially avoid the issues of managing a huge amount of data:
+  * 44 reference sequences (see above). 
+  * 207 sequences acquired from GenBank (some of these are also my reference sequences) using the BLAST tool (https://blast.ncbi.nlm.nih.gov/Blast.cgi). 
+* I will add to this dataset later with methods as follows:
+  * First, I will search for sequences assigned to the genus *Stereum*, and discard any that are mislabeled (similarity of less than 85% compared to my Stereum sequences) or too short (<375 bases).
+  * Then, I will use NCBI GenBank BLAST to locate sequences that are similar to those I already have (my outgroup *Xylobolus subpileatus* is 83-78% similar, so I think 85% will be a good cutoff). 
+##### Reflections
+My personal challenge with this project is that I have somehow avoided learning programming up to this point. I have a clear idea of what I want to achieve, but run into all kinds of obstacles while figuring out how to do it. Despite that, I am excited to finish this project and learn new skills, even if it takes me a while to get up to speed and start to develop some kind of programming literacy. 
```

Comment:
a good strategy to learning a seemingly new and challenging set of skills is to break it up into smaller pieces that you can more easily conquer. so I would suggest you first try your best to outline the steps, making sure that you knows *exactly* what to do for the first one or two steps, at least in theory, so you can get started.